.github/workflows/ensure-pr-labels.yaml

```
@@ -0,0 +1,61 @@
+name: Check PR Area Label
+
+on:
+  pull_request:
+    types: [opened, edited, labeled, unlabeled]
+
+permissions:
+  pull-requests: write
+  contents: read
+
+jobs:
+  manage-area-labels:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check and manage PR labels
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          AREA_COMMENT: "Please add an appropriate `area/*` label (e.g., area/feature, area/bug) to this pull request."
```

```suggestion
          AREA_COMMENT: "Please add an appropriate `kind/*` label (e.g., kind/feature, kind/bug) to this pull request."
```
Area should refer to the part of the project the PR touches.

---

@kkaempf could you please clarify what is preferred?

---

> @kkaempf could you please clarify what is preferred?

This: Area should refer to the part of the project the PR touches.

So `kind/bug` is ok, `area/bug` is not. (and `kind/feature` should actually be `kind/enhancement` for consistency with other repos).

`kind` should be obvious by looking at [existing kind labels](https://github.com/rancher/turtles-docs/labels) :wink:

P.S.: Github recently added a `Type` setting in issues which is somewhat overlapping with kind. However, it doesn't cover all kinds we have currently defined.

---

should the workflow check for `kind/*` in PRs and `area/*` in issues?

---

Looking at the upstream repos, they are both used in both PRs/issues IMO